internal/caching/cache_lazy_load_members.go

```diff
@@ -14,6 +14,7 @@ type lazyLoadingCacheKey struct {
 type LazyLoadCache interface {
 	StoreLazyLoadedUser(device *userapi.Device, roomID, userID, eventID string)
 	IsLazyLoadedUserCached(device *userapi.Device, roomID, userID string) (string, bool)
+	Invalidate(device *userapi.Device, roomID, userID string)
```

Comment:
Please name this `InvalidateLazyLoadedUser` so that the interfaces can overlap if we need to define another invalidate function on another cache later.